doc/mgr/dashboard.rst

```diff
@@ -259,6 +270,18 @@ commands::
 
   $ ceph dashboard ac-user-create <username> <password> administrator
 
+Accessing the Dashboard
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can now access the dashboard using your (JavaScript-enabled) web browser, by
+pointing it to any of the host names or IP addresses and the selected TCP port
+where a manager instance is running: e.g., ``httpS://<$IP>:<$PORT>/``.
```

Comment:
I'm torn with the spelling of `httpS://<$IP>:<$PORT>/`. Isn't there a standard for doing this? Can a regex used here? To me it should look like `http(s)?://<$IP>:<$PORT>/`, `http[s]://<$IP>:<$PORT>/` or something else. Any ideas?

Reply:
I went with `http(s)` now. Thanks!